docs/source-fabric/index.rst

```diff
@@ -82,20 +104,56 @@ This makes it easier to develop and debug your PyTorch code as you gradually add
 Fabric provides important tools to remove undesired boilerplate code (distributed, hardware, checkpoints, logging, ...), but leaves the design and orchestration fully up to you.
 
 
-----
+.. raw:: html
 
-************
-Installation
-************
+   <hr class="docutils" style="margin: 50px 0 50px 0">
 
-Fabric ships directly with Lightning. Install it with
+Get Started
+-----------
 
-.. code-block:: bash
+.. raw:: html
 
-    pip install lightning
+    <div class="tutorials-callout-container">
+        <div class="row">
 
-For alternative ways to install, read the :doc:`installation guide <fundamentals/installation>`.
+.. Add callout items below this line
```

Comment:
Shouldn't the titles here match the ones on the cards? https://lightning.ai/docs/fabric/stable/levels/basic.html

Reply:
good call, was checking if we can just import the basic but seems that links which are relative fails as index and levels/basic is in different structure level :(

Reply:
so we can just copy-paste text but nowt be sure if later the text diverge again...

Reply:
FYI, did the copy paste